cves/kernel/CVE-2020-29370.yml

```diff
@@ -131,10 +138,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: no identifiable unit tests in the tools selftest folder and the file itself is mostly referenced as comments or in config files.
+  fix: false
+  fix_answer: since there were no unit tests, none were fixed. 
```

Comment:
Just to confirm, this means that no tests were added in the fix commit either?

Reply:
yes, I couldnt find any within the commits that tested the methods / file.